region/brinstar/pink.json

```diff
@@ -2428,6 +2428,18 @@
                     }}
                   ],
                   "note": "Doable without a short charge, since there are 32 tiles (plus one open end) to charge it."
+                },
+                {
+                  "name": "Shinespark Come in Charged",
+                  "notable": false,
+                  "requires": [
+                    {"canComeInCharged": {
+                      "fromNode": 1,
+                      "framesRemaining": 5,
```

Comment:
Looks like around 45frames to me.  You could pretend 5frames reaches the raised runway to start another spark 🏇

Reply:
It felt small, so I put in a small number. Didn't realize how far off it was. Now I know how to find the values. I could do it in 27 with a little run speed through the door, so I put 40.